cli/command/plugin/inspect.go

```diff
@@ -37,20 +34,8 @@ func newInspectCommand(dockerCli *command.DockerCli) *cobra.Command {
 func runInspect(dockerCli *command.DockerCli, opts inspectOptions) error {
```

Comment:
Thanks @anusha-ragunathan. The help output and the related docs has been updated.